### ISSUE TITLE
Accessibility Issue - custom button opening the sidebar menu cannot be activated with keyboard

### ISSUE THREAD

**Issue by natnael2012** [learningequality/ka-lite#3346]
Branch – Develop
Expected behavior – All buttons should be focus able and has to handle keyboard events in order to be usable by both keyboard only users and users using screen reader.
Current behavior – Custom button on the left opening the sidebar menu cannot be activated with keyboard.

**Comment by MCGallaspy** [comment]
Is this too ambitious for 0.14?

**Comment by rtibbles** [comment]
Given that @radinamatic will be implementing most of this, and her GSoC term does not start until June, I would say, yes.

**Comment by radinamatic** [comment]
Unless my coding skills get waaaay better in a matter of days by some KA Lite's own miracle, yeah, I'd say it's too ambitious... ;)

**Comment by rtibbles** [comment]
I assume this will require the addition of text to describe the button for screen readers. Punting.

**Comment by radinamatic** [comment]
I could only wish this is the sole keyboard navigation issue KA Lite had... :sob:

**Comment by benjaoming** [comment]
Opening and navigating the menus is working fine with arrow navigation on Chrome.
